README.md

```diff
@@ -109,6 +109,11 @@ Extra Java options for the Jenkins launch command configured in the init file ca
 
 Changes made to the Jenkins init script; the default set of changes set the configured URL prefix and add in configured Java options for Jenkins' startup. You can add other option/value pairs if you need to set other options for the Jenkins init file.
 
+    jenkins_use_proxy: yes
+
+Whether or not to use the system supplied proxy when fetching information via
+`get_url`.
```

Comment:
This should be on one line (for markdown files, I don't break at 80 cols).

Reply:
Markdown files are generally rendered in a UI or preview tool where it's better to let your editor/GitHub/etc. control the text flow, so I prefer for files which are more end-user-facing to have freeform paragraphs (which are also easier to edit in long form) than README files which might be intended only for use in editors or on the CLI.

Reply:
@geerlingguy thanks for that explanation 👍 for me wide line lengths are going to be a problem in any file where you might want to view a side-by-side diff (like what I am looking at here on github.com), regardless of the type